internal/fuzz/path.go

```diff
@@ -91,13 +91,28 @@ func PerformPathFuzz(ctx context.Context, target string, pathlist string, respon
 		return PathReport{}, nil
 	}
 
+	// 8. Profile the base URL if ignorebase is true
+	baseProfile := HTTPResponseProfile{}
+	if ignorebase {
+		baseProfile, err = profileBaseURL(target)
+		if err != nil {
+			return PathReport{}, err
+		}
+	}
+
 	report := PathReport{
 		Target: target,
 		URLs:   []URLDetails{},
 		Errors: []string{},
 	}
 
 	for _, result := range customOutput.CurrentResults {
+		if ignorebase && baseProfile.StatusCode == 200 {
+			// ffuz seems to report an extra line for every response, so we need to check for both the base profile lines and the base profile lines + 1
+			if result.ContentLength == int64(baseProfile.Size) && (result.ContentLines == int64(baseProfile.Lines)+1 || result.ContentLines == int64(baseProfile.Lines)) {
```

Comment:
I worry we're just checking the line count and length, not the content itself. It would be more expensive but perhaps we could do a quick hash to check content?